### ISSUE TITLE
Can you change method from OCR to API?

### ISSUE THREAD

**Issue by PoonamChohdhary1991** [sushant10/HQ_Bot#65]
This method is very slow and unreliable. It takes almost 20 seconds to run and also not accurate and sometimes it is not able to read questions properly. 
Also if you do so can you please provide it for brain baazi , loco and iq trivia

**Comment by 0bytedna** [comment]
You can use the same script for any trivia game, provided you set the dimensions of the screen properly.

**Comment by TiNivla** [comment]
Yeah im having the same issue, especially with accuracy.  The ocr doesn't read the text all the accurate either.
